src/app/editor.js

```diff
@@ -23,7 +23,9 @@ var css = csjs`
     width             : 100%;
   }
 `
-document.querySelector('#editor-container').className = css['editor-container']
+setTimeout(_ => {
```

Comment:
yes that's what this refactoring is all about.
The markup is currently set up in a way that makes it hard to add the console.
That's why i'm doing the refactoring.

The lines you wrote above is exactly what i want to make happen, even though i personally think the `.init()` function is what should happen in the `App(...)` constructor